README.md

```diff
@@ -180,7 +180,33 @@ engine.addActions({thread: commentThreadActions})
 engine.addListener((store) => document.body.innerHTML = store.thread.join('<br />'))
 ```
 
-Like `addActions`, you can add as many listeners as you want, each will be called with the new store.
+Like `addActions`, you can add as many listeners as you want by calling `addListener` multiple times. Each will be called with the new store.
+
+#### Listener Arguments
+Along with the store and actions, listeners also recieve the name of the action that was called, and the argument it was called with. With these, you can use listeners to debug what is happening in hover-engine. In the example below, we log the action and the new values in the store.
+
+```javascript
+const debugListener = (store, actions, actionName, actionArguments) => {
+  console.log(actionName, actionArguments, '->', store)
+}
+```
+
+### `notifyListeners(action)`
```

Comment:
## Documentation
We never really documented notifyListeners... There's enough tests around it that I feel okay including it.